x-pack/docs/en/security/authentication/saml-guide.asciidoc

```diff
@@ -782,17 +775,19 @@ example below:
 
 [source, yaml]
 ------------------------------------------------------------
-xpack.security.authc.providers: [saml, basic]
+xpack.security.authc.providers:
+  saml.saml1:
+    order: 0
+    realm: "saml1"
+  basic.basic1:
+    order: 1
 ------------------------------------------------------------
 
-The order is important - this will _initiate_ SAML authentication for
-unauthenticated users, but will _accept_ basic authentication.
-
-If {kib} is configured in this way, then users who wish to login with a
-username and password, can do so by directly accessing the `/login` page in
-{kib}. This login will not use SAML credentials, and will rely on one of the
-other security realms within {es}. Only users who have a username and password
-for a configured {es} authentication realm will be able to login via this page.
+If {kib} is configured in this way, users are presented with a choice
+at the Login Selector UI. They can provide a username and password, and rely on one
```

Comment:
Trying to make it clear that "in this sentence we're describing just two options - either SAML or not-SAML":
```suggestion
at the Login Selector UI. They log in with SAML or they provide a username and password and rely on one
```

Reply:
Thanks! Committed your suggestions.